src/chemcalculator/chemcalculator.py

```diff
@@ -152,11 +152,20 @@ def moles_grams_converter(formula, mass, converter):
     Examples
     --------
     >>> moles_grams_converter("H2O", 0.05555, "moles")
-    1.000748804
+    1.000
 
     >>> moles_grams_converter("H2O", 18.01528, "grams")
-    1
+    1.000
     """
+    grams_per_mole = compute_mass()
```

Comment:
compute mass has an input argument
```suggestion
    grams_per_mole = compute_mass(formula)
```